lib/archethic/mining/standalone_workflow.ex

```diff
@@ -42,11 +42,15 @@ defmodule Archethic.Mining.StandaloneWorkflow do
 
   def init(arg) do
     tx = Keyword.get(arg, :transaction)
+    welcome_node = Keyword.get(arg, :welcome_node)
```

Comment:
I recommend `Keyword.fetch!/2` since it's not optional at all.

Reply:
👍